rubin_sim/scheduler/example/example_scheduler.py

```diff
@@ -1169,18 +1030,27 @@ def generate_twi_blobs(
     return surveys
 
 
+def ddf_surveys(detailers=None, season_unobs_frac=0.2, euclid_detailers=None):
+    obs_array = generate_ddf_scheduled_obs(season_unobs_frac=season_unobs_frac)
+
+    euclid_obs = np.where(
+        (obs_array["note"] == "DD:EDFS_b") | (obs_array["note"] == "DD:EDFS_a")
+    )[0]
+    all_other = np.where(
+        (obs_array["note"] != "DD:EDFS_b") & (obs_array["note"] != "DD:EDFS_a")
+    )[0]
+
+    survey1 = ScriptedSurvey([], detailers=detailers)
+    survey1.set_script(obs_array[all_other])
+
+    survey2 = ScriptedSurvey([], detailers=euclid_detailers)
+    survey2.set_script(obs_array[euclid_obs])
+
+    return [survey1, survey2]
+
+
 def ecliptic_target(nside=32, dist_to_eclip=40.0, dec_max=30.0, mask=None):
-    """Generate a target map for the area around the ecliptic
-
-    Paramters
-    ---------
-    dist_to_eclip : float (40)
-        How far to extend from the ecliptic (degrees). default 40.
-    dec_max : float (30)
-        The maximum declination to extend to (degrees). default 30.
-    mask : np.array (None)
-        A HEALpix map that the result is multiplied by (default None)
-    """
+    """Generate a target_map for the area around the ecliptic"""
```

Comment:
Why drop the docstring for the parameters? If it's private enough that the docstring isn't needed, we should rename the function with an underscore prefix (`_ecliptic_target`).